### PR TITLE
Ruby 2.3.1

### PR DESCRIPTION
**Pull request by jessieay**
* Upgrading is fun <3

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,8 +32,6 @@ group :test do
   gem "codeclimate-test-reporter", require: nil
   gem 'codeclimate_batch', require: nil
   gem 'dotenv'
-  gem 'webmock'
```
Comment (gemfarmer) [inline Gemfile]: @jessieay were these supposed to be removed?

Comment (jessieay) [inline Gemfile]: they're duplicates :) Bundler notified me that they were listed twifce

Comment (harrisj) [inline Gemfile]: Thank you!

**Comment by harrisj** [comment]
LGTM. Merging

**Pull request merged**